src/main/java/se/isselab/HAnS/metrics/FeatureMetrics.java

```diff
@@ -81,6 +84,20 @@ public FeatureMetrics(FeatureModelFeature feature, int tanglingDegree, int scatt
         this.scatteringDegree = scatteringDegree;
     }
 
+    /**
+     * Generate Feature Metrics with Feature, Tangling Degree and Scattering Degree
+     *
+     * @param feature         FeatureModelFeature
+     * @param maxNestingDepth         maximum nesting depth of a {@link FeatureModelFeature}
+     * @param minNestingDepth         minimum nesting depth of a {@link FeatureModelFeature}
+     * @param avgNestingDepth         average nesting depth of a {@link FeatureModelFeature}
+     */
+    public FeatureMetrics(FeatureModelFeature feature, int maxNestingDepth, int minNestingDepth, int avgNestingDepth) {
```

Comment:
Because otherwise I get errors from **ScatteringDegreeBackground** and **TanglingDegreeBackground**, since they use _featureMetrics = new FeatureMetrics(feature, -1, scatteringDegree);_ or _super.featureMetrics = new FeatureMetrics(feature, tanglingDegree, -1);_ and I wasn't sure whether I should change the constructor and write e.g. _featureMetrics = new FeatureMetrics(feature, -1, scatteringDegree, null, null, null);_ or leave it as it is